lms/djangoapps/ora_staff_grader/mock/data/ora_metadata.json

```diff
@@ -0,0 +1,80 @@
+{
+    "block-a": {
+        "name": "Individual ORA",
+        "prompts": [
+            {
+                "description": "<p>Enter a text/files response.</p>"
+            }
+        ],
+        "type": "individual",
+        "rubricConfig": {
+            "feedback": "optional",
+            "feedbackPrompt": "How would you grade this response?",
```

Comment:
This is probably a different ticket but I don't see any translation of the backend snake_case keys to these camelCase ones.

Reply:
There is no translation, this is just the data the BFF would return.

Reply:
I don't think it is, because the xblock handler returns snake case, and the BFF just hands that back from the handler

Reply:
In that case, please update the contracts page and ping me when when it's ready. That is source of truth and what I'm working from so 🤷

Reply:
Ooh, actually. I see the rubric data shape in the contracts doc. But the contracts doc also shows the camelCased fields being used by the frontend. Need to either (a) ticket that field translation or (b) tell frontend folks to use the snake_case names. Either way, contracts doc should be updated.

Thoughts @jansenk / @muselesscreator ?

Reply:
Let's leave this as-is since it matches the style we're going for, but add a ticket for doing a BFF-level translation

Reply:
Sorry for leaving this comment for seven hours lmao. "Busy" day

Reply:
Currently I've reverted to pass-through. If decided we want translation, I'll add a ticket for updating both mock and real versions